packages/playwright-test/src/expect.ts

```diff
@@ -48,9 +48,7 @@ import type { Expect, TestError } from './types';
 import matchers from 'expect/build/matchers';
 import { currentTestInfo } from './globals';
 import { serializeError } from './util';
-import StackUtils from 'stack-utils';
-
-const stackUtils = new StackUtils();
+import { captureStackTrace } from 'playwright-core/lib/utils/stackTrace';
```

Comment:
Suggested wrapper in `playwright-test/src/util` that removes notion of `expect` from core entirely.

```js
function captureStackTrace() {
  const stack = require('...core').captureStackTrace();
  for (let i = stack.frames.length - 1; i >= 0; i--) {
    if (stack.frames[i] is EXPECT) {
      stack.frames.splice(i, 1, 0);
      stack.frameTexts.splice(i, 1, 0);
    }
  }
  return stack;
}
```